mod-users-bl/src/main/resources/domain/mod-users-bl/features/users-bl.feature

```diff
@@ -77,3 +74,89 @@ Feature: Test user business logic
     When method POST
     Then status 201
     And match responseHeaders contains { 'x-okapi-token': '#present' }
+
+  Scenario: Return a composite object for the currently logged in user
+
+    Given path 'bl-users/_self'
+    When method GET
+    Then status 200
+    And match response.user.active == true
+    And match response.user.personal.firstName == 'Admin'
+    And match response.user.username == 'test-admin'
+
+  Scenario: Return a composite object for the currently logged in user by username
+
+    Given path 'bl-users/by-username/' + 'test-admin'
+    When method GET
+    Then status 200
+    And match response.user.active == true
+    And match response.user.personal.firstName == 'Admin'
+    And match response.user.username == 'test-admin'
```

Comment:
What is special about the currently logged in user? I imagine the admin user can access any user via this API.

Reply:
yes , i have changed scenario description.